core/badge-urls/make-badge-url.js

```diff
@@ -62,9 +62,6 @@ function staticBadgeUrl({
   namedLogo,
   format = 'svg',
 }) {
-  if (!label || !message) {
```

Comment:
Apologies as I'm _pretty_ sure I asked this question before recently (having serious deja vu if not), but for my own education why do we no longer need to require one of these (I'm assuming they were needed before)?

Reply:
😁It was here: https://github.com/badges/shields/pull/2869#discussion_r251168779

I had to change it again here because this badge has _no message_:

<img width="834" alt="screen shot 2019-01-29 at 4 45 16 pm" src="https://user-images.githubusercontent.com/1487036/51945806-47c3ad80-23e5-11e9-8420-da634b5b0fd6.png">

Reply:
Thanks for helping me on this (twice 😉). Committed to memory this time!